pxtblocks/fields/field_note.ts

```diff
@@ -228,13 +228,13 @@ namespace pxtblockly {
          * refresh the text to get back into a valid state.
          **/
         protected refreshText() {
-            this.setText(this.getText());
```

Comment:
the reason the set text was here was that there were a few situations in which the display wouldn't update properly / actually rerender when things have been swapped; I believe the main one was after typing in a note and committing it by pressing enter (or clearing the entry and pressing enter) -- I wouldn't be surprised if that was fixed elsewhere as there wasn't a clear reason why it was broken, but if you could double check that works that would be good~

Reply:
yep, i tested typing for both named and non-named notes when upgrading the field, but the build links above are current if you want to double check!

Reply:
Ahh, I think the overlap with `for` when opening might have been fixed with resetting the text to get the updated size before and that can be fixed later, everything else seems fine~